src/Knp/Rad/ResourceResolver/EventListener/ResourcesListener.php

```diff
@@ -105,4 +112,12 @@ protected function castParameter($parameter)
             }
         }
     }
+
+    protected function isNullValue($resource)
+    {
+        return null === $resource ||
+            (is_array($resource) && empty($resource)) ||
+            ($resource instanceof \Countable && 0 === count($resource))
```

Comment:
You can simply check if `empty($resource)`, because empty will check all `null` "possibilities" (null, empty array, empty string, int/float with 0 as value, declared but not assigned variables).